test/tile-test.js

```diff
@@ -18,8 +18,64 @@ tape("tile", function(test) {
   test.equal(tiles.translate[0], -1.68359375);
   test.equal(tiles.translate[1], -5.08203125);
   test.equal(tiles.length, 15);
-  test.deepEqual(tiles[0], [ 1, 5, 4 ]);
-  test.deepEqual(tiles[1], [ 2, 5, 4 ]);
-  test.deepEqual(tiles[2], [ 3, 5, 4 ]);
+
+  var d = [1, 5, 4];
+  d.x = 256;
+  d.y = 5 * 256;
+  test.deepEqual(tiles[0], d);
```

Comment:
I recommend creating a helper function like this:

```js
function tile(x, y, z, tx, ty) {
  var d = [x, y, z];
  d.x = tx;
  d.y = ty;
  return d;
}
```

Then you can do:

```js
test.deepEqual(tiles[0], tile(1, 5, 4, 256, 5 * 256));
```

And even better:

```js
test.deepEqual(tiles, [
  tile(1, 5, 4, 1 * 256, 5 * 256),
  tile(2, 5, 4, 2 * 256, 5 * 256),
  tile(3, 5, 4, 3 * 256, 5 * 256),
  …
]);
```